### DISCUSSION TITLE
Software Engineering Survey on GitHub Discussions.

### DISCUSSION THREAD

**Discussion by raux** [prisma/prisma#3276] [Q&A] [answered]
Do you find GitHub discussions useful as a communication channel? Maybe not? 
Hello Developers! I am Raula Gaikovina Kula, a Software Engineering researcher
https://raux.github.io/

Together with a team of researchers, we are interested in your thoughts about
using the GitHub Discussions Beta. We noticed that your project is currently using.

We have developed a simple 3 question survey that could take up to 5 mins to complete.
https://forms.gle/peBxiio8u99QvKMk6

Thank you in advance for your time and apologize if this is not a proper use of this channel.

@raux
Raula, PhD on behalf of Hideaki, Sebastian, Nicole and Christoph

**Comment by nikolasburk** [comment, marked as answer]
Looks great! I just submitted a response @raux, hope that helps 🙌  Let me know if you have any follow up questions 🙂

**Reply by raux** [reply]
thank you so much for the feedback. I will definitely keep you in the loop with the results we get. 

Raula

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
